examples/stm32/nucleo-f429zi-freertos-mip/Makefile

```diff
@@ -0,0 +1,33 @@
+CFLAGS  ?=  -W -Wall -Wextra -Werror -Wundef -Wshadow -Wdouble-promotion \
+            -Wformat-truncation -fno-common -Wconversion \
+            -g3 -Os -ffunction-sections -fdata-sections \
+            -mcpu=cortex-m4 -mthumb -mfloat-abi=hard -mfpu=fpv4-sp-d16 \
+            -I. -Icmsis -I../../../ $(EXTRA_CFLAGS)
+LDFLAGS ?= -Tlink.ld -nostartfiles -nostdlib --specs nano.specs -lc -lgcc -Wl,--gc-sections -Wl,-Map=$@.map
+SOURCES = main.c startup.c syscalls.c ../../../mongoose.c
+
+FREERTOS_VERSION ?= V10.5.0
+FREERTOS_REPO ?= https://github.com/FreeRTOS/FreeRTOS-Kernel 
+
+build example: firmware.elf
+
+firmware.elf: FreeRTOS-Kernel $(SOURCES) 
+	arm-none-eabi-gcc -o $@ $(SOURCES) $(CFLAGS) $(LDFLAGS) \
```

Comment:
This requires having the ARM toolchain installed. Why not keep using the docker version ? (Yes, those nasty warnings, but we'll get rid of them, eventually...)

Reply:
Docker added, yes